docs/intro/tutorial.rst

```diff
@@ -44,6 +44,36 @@ as well as the `suggested resources in the learnpython-subreddit`_.
 .. _Learn Python 3 The Hard Way: https://learnpythonthehardway.org/python3/
 .. _suggested resources in the learnpython-subreddit: https://www.reddit.com/r/learnpython/wiki/index#wiki_new_to_python.3F
 
+Core Scrapy Components and Their Roles
```

Comment:
Not sure if this belongs in the tutorial (also some of these phrases are incorrect).